### ISSUE TITLE
Error build Linux 

### ISSUE THREAD

**Issue by giaphong** [nextgis/nextgisqgis#78]
-- Not found ZLIB in packages. Try look in system.
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11") 
-- Not found SQLite3 in packages. Try look in system.
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Checking for module 'sqlite3'
--   Found sqlite3, version 3.22.0
-- Found SQLITE3: /usr/lib/x86_64-linux-gnu/libsqlite3.so  
-- Not found EXPAT in packages. Try look in system.
-- Found EXPAT: /usr/lib/x86_64-linux-gnu/libexpat.so (found version "2.2.5") 
-- Not found GEOS in packages. Try look in system.
-- Found GEOS: /usr/lib/x86_64-linux-gnu/libgeos_c.so (found version "3.7.0") 
-- Not found PROJ in packages. Try look in system.
-- Found PROJ: /usr/lib/x86_64-linux-gnu/libproj.so (found version "4.9.3") 
-- Found GDAL in package repository:  (found version "2.4.0")
-- Not found Qt4 in packages. Try look in system.
-- Looking for Q_WS_X11
-- Looking for Q_WS_X11 - found
-- Looking for Q_WS_WIN
-- Looking for Q_WS_WIN - not found
-- Looking for Q_WS_QWS
-- Looking for Q_WS_QWS - not found
-- Looking for Q_WS_MAC
-- Looking for Q_WS_MAC - not found
-- Could NOT find Qt4 (missing: QT_QTWEBKIT_INCLUDE_DIR QT_QTWEBKIT_LIBRARY) (found version "4.8.7")
CMake Error at cmake/FindAnyProject.cmake:190 (message):
  Qt4 is required in qgis!
Call Stack (most recent call first):
  CMakeLists.txt:178 (find_anyproject)

**Comment by BishopGIS** [comment]
According to error message the is no libqtwebkit-dev in your system.
The list of dependencies are here: https://github.com/nextgis/ppa/blob/master/nextgisqgis/debian/control#L7-L38

**Comment by giaphong** [comment]
Error ? 
![image](https://user-images.githubusercontent.com/34709818/69853469-8fdb8700-12b9-11ea-8f69-b6de6bca6084.png)

**Comment by BishopGIS** [comment]
Now build script cannot found libqca2-dev.

**Comment by giaphong** [comment]
Hi @BishopGIS ,
I installed libqca2-dev on my system,
but i still get errors 

![image](https://user-images.githubusercontent.com/34709818/69859786-6cb7d400-12c7-11ea-94da-a108bc33ec4e.png)

Pls, help

**Comment by giaphong** [comment]
![image](https://user-images.githubusercontent.com/34709818/69862832-eeaafb80-12cd-11ea-8000-af99811b6a86.png)

**Comment by BishopGIS** [comment]
Let me try to reproduce this error.

**Comment by giaphong** [comment]
@BishopGIS  Pls, help 
cmake -DWITH_EXPAT=ON -DWITH_GDAL=ON -DWITH_GEOS=ON -DWITH_GSL=ON -DWITH_LibXml2=ON \
    -DWITH_PostgreSQL=ON -DWITH_PROJ4=ON -DWITH_Qca=ON -DWITH_Qscintilla=ON -DWITH_Qwt=ON \
    -DWITH_SpatialIndex=ON -DWITH_Spatialite=ON -DWITH_SQLite3=ON -DWITH_SIP=ON \
    -DWITH_Qt4=ON -DWITH_BINDINGS=ON -DWITH_PyQt4=ON -DWITH_Qsci=ON -DWITH_ZLIB=ON -DWITH_BINDINGS=ON
ngqgis version 19.11.0
-- Not found ZLIB in packages. Try look in system.
-- Found SQLite3 in package repository: /usr/lib/x86_64-linux-gnu/libsqlite3.so (found version "")
-- Not found EXPAT in packages. Try look in system.
-- Not found GEOS in packages. Try look in system.
-- Not found PROJ4 in packages. Try look in system.
-- Not found GDAL in packages. Try look in system.
-- Not found Qt4 in packages. Try look in system.
-- Found Qca in package repository: /usr/lib/x86_64-linux-gnu/libqca.so (found version "")
-- Found Qscintilla in package repository: /usr/lib/libqscintilla2_qt4.so (found version "")
-- Found Qwt in package repository: /usr/lib/libqwt-qt4.so (found version "")
-- Found Spatialite in package repository: /usr/lib/x86_64-linux-gnu/libspatialite.so (found version "")
-- Found SpatialIndex in package repository: /usr/lib/x86_64-linux-gnu/libspatialindex.so (found version "")
-- Found PostgreSQL in package repository: /usr/lib/x86_64-linux-gnu/libpq.so (found version "11.5 (Ubuntu 11.5-0ubuntu0.19.04.1)")
-- Not found OCI in packages. Try look in system.
-- Could not find OCI
CMake Error at cmake/FindAnyProject.cmake:180 (message):
  OCI is required in qgis!
Call Stack (most recent call first):
  src/CMakeLists.txt:69 (find_anyproject)


-- Configuring incomplete, errors occurred!
See also "/home/giaphong/Desktop/nextgisqgis/CMakeFiles/CMakeOutput.log".
See also "/home/giaphong/Desktop/nextgisqgis/CMakeFiles/CMakeError.log".

**Comment by BishopGIS** [comment]
I added some fixes to build on Ubuntu 18.04. The steps to build are following.

```bash
sudo apt install git cmake build-essential
mkdir work && cd work
git clone https://github.com/nextgis/nextgisqgis.git
cd nextgisqgis
mkdir build && cd build
sudo add-apt-repository ppa:nextgis/ppa && sudo apt update
sudo apt install libexpat1-dev libgdal-dev libgeos-dev libgsl-dev libpq-dev libproj-dev libqt4-dev libqca2-dev libsqlite3-dev libspatialite-dev libqwt5-qt4-dev libqscintilla2-qt4-dev libqtwebkit-dev libspatialindex-dev libngstd-framework4-dev python-sip-dev python-qt4-dev python-qt4 pyqt4-dev-tools python-qscintilla2 pyqt4.qsci-dev python-six python-raven python-contextlib2 libopencv-core-dev libopencv-highgui-dev libopencv-ml-dev libclntsh-dev
cmake -DWITH_EXPAT=ON -DWITH_GDAL=ON -DWITH_GEOS=ON -DWITH_GSL=ON -DWITH_LibXml2=ON -DWITH_PostgreSQL=ON -DWITH_PROJ=ON -DWITH_Qca=ON -DWITH_Qscintilla=ON -DWITH_Qwt=ON -DWITH_SpatialIndex=ON -DWITH_Spatialite=ON -DWITH_SQLite3=ON -DWITH_SIP=ON -DWITH_Qt4=ON -DWITH_BINDINGS=ON -DWITH_PyQt4=ON -DWITH_Qsci=ON -DWITH_ZLIB=ON -DWITH_OCI=ON ..
cmake --build . --config Release -- -j4
```
